preprocess.py

```diff
@@ -4,6 +4,7 @@
 t2s = OpenCC('t2s').convert
 
 os.system('wget -nc https://raw.githubusercontent.com/rime/rime-cantonese/5b6d334/jyut6ping3.dict.yaml')
```

Comment:
How much will the results be affected if we eliminate the downstream file and grab all files from upstream? Presuming we are going with this, let’s make upstream a submodule instead.

Reply:
I think submodule would be hard to maintain

Reply:
How will? Isn’t a submodule just an SHA, as if we manually amend the link regularly in this file?

Reply:
I see. Then it may not be a problem